### PR TITLE
Enable invisible history by default plus cleanup

### PR DESCRIPTION
**Pull request by SergeyGaluzo**
Fixes
https://microsofthealth.visualstudio.com/Health/_workitems/edit/109167/
https://microsofthealth.visualstudio.com/Health/_workitems/edit/109165/

### CODE REVIEW

**Comment by SergeyGaluzo** [comment]
azp /run

File: src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/MergeResourcesBeginTransaction.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿--DROP PROCEDURE dbo.MergeResourcesBeginTransaction
 GO
-CREATE PROCEDURE dbo.MergeResourcesBeginTransaction @Count int, @TransactionId bigint = 0 OUT, @SurrogateIdRangeFirstValue bigint = 0 OUT, @SequenceRangeFirstValue int = 0 OUT, @HeartbeatDate datetime = NULL -- TODO: Remove @SurrogateIdRangeFirstValue
+CREATE PROCEDURE dbo.MergeResourcesBeginTransaction @Count int, @TransactionId bigint OUT, @SequenceRangeFirstValue int OUT, @HeartbeatDate datetime = NULL -- TODO: Remove @SurrogateIdRangeFirstValue
```
Comment (LTA-Thinking) [inline src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/MergeResourcesBeginTransaction.sql:3]: The TODO comment isn't needed anymore

Comment (SergeyGaluzo) [reply src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/MergeResourcesBeginTransaction.sql]: Fixed

**Review by LTA-Thinking — DISMISSED** [review]

**Review by feordin — APPROVED** [review]

**Pull request merged**